website/highlights/2020-07-09-add-musl-and-glibc-support-to-install-sh.md

```diff
@@ -0,0 +1,52 @@
+---
+last_modified_on: "2020-07-14"
+$schema: "/.meta/.schemas/highlights.json"
+title: "glibc enhancements"
+description: "If your Linux uses glibc, Vector will too."
+author_github: "https://github.com/hoverbear"
+hide_on_release_notes: false
+pr_numbers: [2969,2518]
+release: "0.10.0"
+tags: ["type: breaking change", "domain: operations"]
+---
+
+As a result of some recent profiling and benchmarking, we determined that builds of Vector targetting `x86_64-unknown-linux-gnu` outperformed builds targetting `x86_64-unknown-linux-musl` under most usage scenarios on supported operating systems.
+
+After some deliberation, we've opted to package and ship glibc builds where it's safe. While this change means Vector is no longer fully static on operating systems which use glibc, it should provide a better user experience.
+
+## Measures we've take to safeguard your deployments
+
+* This change **only affects** x86 platforms.
+* This change **affects** `.deb` and `.rpm` packages.
+* This change **affects** Vectors installed via `https://sh.vector.dev`.
+* This change **does not affect** any [archives][urls.vector_download] you may already be using. We now publish archives with the `gnu` prefix that contain glibc builds. *Musl builds are untouched as `musl` still.*
+* This change **does not affect** non-Linux platforms.
+
+If you've fought with glibc before, you've probably got a burning question:
+
+> Which version do we support? 🕵️‍♀️
+
+**Don't worry. We have your back. 🤜🤛** We're using a base of CentOS 7, which means new Vector glibc builds will support all the way back to **glibc 2.17**. If that's still too new for your machines, please keep using the Musl builds. (Also, [Let us know!][urls.new_bug_report])
```

Comment:
It might be worth adding a note about how old 2.17 is. I'm not sure how many users will know their glibc version off the top of their heads, but something about "N years old!" might help put them at ease.